src/main/java/use_case/leaderboard_as_chart/LeaderboardAsChartInputData.java

```diff
@@ -0,0 +1,13 @@
+package use_case.leaderboard_as_chart;
+
+public class LeaderboardAsChartInputData {
+    private final String requestType;
+
+    public LeaderboardAsChartInputData(String requestType) {
+        this.requestType = requestType;
+    }
+
+    public String getRequestType() {
+        return requestType;
+    }
```

Comment:
```suggestion
    }
    
    public int getLeaderboardEntryCount() {
        return leaderboardEntryCount;
    }
```

You need a **getter** for the argument `leaderboardEntryCount` so that the `leaderboardInteractor` can access the instances.